src/components/tooltip/bl-tooltip.ts

```diff
@@ -142,25 +133,29 @@ export default class BlTooltip extends LitElement {
     }
   }
 
-  render(): TemplateResult {
-    const classes = classMap({
-      tooltip: true,
-      visible: this._visible,
-    });
-
+  private triggerTemplate() {
     return html`<slot
-        class="trigger"
-        name="tooltip-trigger"
-        aria-describedby="tooltip"
-        @focus=${{ handleEvent: () => this.show(), capture: true }}
-        @blur=${{ handleEvent: () => this.hide(), capture: true }}
-        @mouseover=${() => this.show()}
-        @mouseleave=${() => this.hide()}
-      >
-      </slot>
-      <div class=${classes}>
-        <slot id="tooltip" role="tooltip" aria-live=${this._visible ? 'polite' : 'off'}></slot>
-        <div class="arrow" aria-hidden="true"></div>
+      class="trigger"
+      name="tooltip-trigger"
+      aria-describedby="tooltip"
+      @focus=${{ handleEvent: () => this.show(), capture: true }}
+      @blur=${{ handleEvent: () => this.hide(), capture: true }}
+      @mouseover=${() => this.show()}
+      @mouseleave=${() => this.hide()}
+    >
+    </slot>`;
+  }
+
+  render(): TemplateResult {
+    return html` ${!this._target ? this.triggerTemplate() : ''}
+      <div class="wrapper">
+        <bl-popover
+          class="tooltip"
+          .target="${!this._target ? this.trigger : this._target}"
```

Comment:
We set target here and also in `show` method. I think one should be dropped. Probably we need to drop this line in favor of `show` method.